components/Form.js

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+export default function Form() {
+  const [title, setTitle] = useState('');
+  const [lyrics, setLyrics] = useState('');
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    setTitle(event.target.title.value.trim());
+    setLyrics(event.target.lyrics.value.trim());
+  }
+
+  return (
+    <>
+      <OutputContainer>
+        <Output>"{title ? title : ''}"</Output>
+        <Output>{lyrics ? lyrics : ''}</Output>
+      </OutputContainer>
+      <InputForm onSubmit={handleSubmit}>
+        Start creating your first idea:
+        <div>
+          <label for="title">Title: </label>
+          <input type="text" id="title" name="title" required />
+        </div>
+        <div>
+          <LyricsField
+            type="text"
+            placeholder="Keep your ideas rolling!"
+            id="lyrics"
+            name="lyrics"
+            required
+          />
+        </div>
+        <Button type="submit">You rock!</Button>
+      </InputForm>
+    </>
+  );
+}
+
+const InputForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 0.9rem;
+  border: 0.1rem solid black;
+  padding: 1.3rem;
+  margin: 2rem;
+`;
+
+const LyricsField = styled.input`
```

Comment:
Same as above: I would move all styling to a seperate StyledComponents file to seperate styling from js-code